build.gradle

```diff
@@ -133,11 +134,32 @@ jacocoTestReport {
 
 project.tasks['sonarqube'].dependsOn jacocoTestReport
 
+pitest {
+  targetClasses = ['uk.gov.hmcts.reform.pcqbackend.*']
+  excludedClasses = [
+    'uk.gov.hmcts.reform.pcqbackend.configuration.*',
+    'uk.gov.hmcts.reform.pcqbackend.controller.*',
+    'uk.gov.hmcts.reform.pcqbackend.Application'
+  ]
+  threads = 10
+  enableDefaultIncrementalAnalysis = true
+  historyInputLocation = ['build/reports/pitest/fastermutationtesting']
+  historyOutputLocation = ['build/reports/pitest/fastermutationtestingoutput']
+  outputFormats = ['XML', 'HTML']
+  timestampedReports = true
+  mutationThreshold = 90
+
```

Comment:
Just a note probate has the following differences
  mutationThreshold = 50
  timeoutConstInMillis = 30000

Do we need a timeout?

Reply:
I took this setting from another project that Satya recommended but can add the settings as per probate. Not sure about the timeout. What is the timeout for?

Reply:
Have added the probate settings now.